packages/gatsby-theme-basic/src/components/Card.js

```diff
@@ -8,18 +8,27 @@ Card.propTypes = {
   as: PropTypes.elementType,
   children: PropTypes.node,
   className: PropTypes.string,
+  contentType: PropTypes.string,
   styles: PropTypes.objectOf(PropTypes.string),
 };
 
 export default function Card({
   as: Component = "article",
   children,
   className,
+  contentType,
   styles = defaultStyles,
   ...restProps
 }) {
   return (
-    <Component className={clsx(styles.component, className)} {...restProps}>
+    <Component
+      className={clsx(
+        styles.component,
+        className,
+        contentType && styles[contentType],
```

Comment:
A generic Card component shouldn’t care about content types. What are you trying to accomplish?

Reply:
trying to add the correct styling for  the old "colored blocks"

Reply:
As we don't have a specific style for "colored blocks", contentType seemed the right one to use. Should I add a new layout type called "colored blocks" instead? That way I can create a different component for it.